taqueria-plugin-flextesa/index.ts

```diff
@@ -49,6 +49,57 @@ Plugin.create(_i18n => ({
             ],
             encoding: "json"
         }),
+        Task.create({
+            task: "typecheck",
+            command: "typecheck <sandboxName> [sourceFiles...]",
```

Comment:
Typo: Remove the trailing period `.` at the end of the error messages

Bug:
It appears that the sandboxName parameter is required in use, this should be optional or removed altogether

Output:
```
taqueria/333 on  557-typechecker-task-JCC 
❯ taq tc taco.tz
There is no sandbox configuration with the name taco.tz.
```

Reply:
The sandbox is actually required (see my response to another comment above, with substring: "Coming from the UX standpoint here:" ).

Reply:
That I understand, but why can't the command use the `default` sandbox specified in `config.json`?

Reply:
Due to a limitation in parsing commands by the Taqueria CLI, I made sandbox mandatory. Otherwise, we'd have optional sandbox and optional source files, and we can't parse this.

Reply:
Ok, will solve with added documentation for now